common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -662,6 +664,7 @@ def _parse_video_xml(cls, xml, id_generator=None):
             id_generator is used to generate course-specific urls and identifiers
         """
         field_data = {}
+        course_id = getattr(id_generator, 'target_course_id', None)
```

Comment:
Nit: Could you move this definition down to just before it's used below? A re-location would make it clearer that the code below doesn't use the `course_id`.